doc/DataPlaneCodeGeneration/AzureSDKCodeGeneration_DataPlane_Quickstart.md

```diff
@@ -58,113 +59,109 @@ sdk\<service name>\<package name>\CHANGELOG.md
 sdk\<service name>\<package name>\<package name>.sln
 ```
 
-- `<service name>` - Should be the short name for the azure service. e.g. deviceupdate
+- `<service name>` - Should be the short name for the azure service. e.g. deviceupdate. It is usually your service name in REST API specifications. For instance, if the API spec is under `specification/storage`, the service would normally be `storage`.
 - `<package name>` -  Should be the name of the shipping package, or an abbreviation that distinguishes the given shipping artifact for the given service. It will be `Azure.<group>.<service>`, e.g. Azure.IoT.DeviceUpdate
 
-### Use Cadl as Input
+### Use Cadl as Input API spec
 
 #### Create Cadl project
   
   You can download existing cadl project from [azure-rest-api-specs](https://github.com/Azure/azure-rest-api-specs) repo or you can follow following steps to create a new cadl project.
```

Comment:
this is misleading as it isn't needed if they use `dotnet build /t:GenerateCode`.  It also isn't part of the "one time setup steps" since they will need to do this every time they generate.

Reply:
Hello @m-nash This is to create cadl project folder  and configure csharp emitter in cadl-project.yml. All the SDK Generation will be put in the `Generate SDK` chapter.

Reply:
Same comment as https://github.com/Azure/azure-sdk-for-net/pull/34058#discussion_r1114822746